lib/mayaUsd/resources/ae/usdschemabase/collectionMayaHost.py

```diff
@@ -313,6 +314,10 @@ def __init__(self):
         self._palette = None
         self._icons = {}
 
+    @property
+    def uiScaleFactor(self) -> float:
+        return float(cmds.mayaDpiSetting(query=True, scaleValue=True))
```

Comment:
The cmd does not exist on OSX, so you should check it exists. And I think you should use the realScaleValue.

`1.0 if not hasattr(cmds, "mayaDpiSetting") else cmds.mayaDpiSetting(query=True, realScaleValue=True)`